src/MLJGLMInterface.jl

```diff
@@ -122,16 +125,27 @@ function prepare_inputs(model, X; handle_intercept=false)
 end
 
 """
-    glm_report(fitresult)
+    glm_report(glm_model, features)
 
-Report based on the `fitresult` of a GLM model.
+Report based on the fitted `LinearModel/GeneralizedLinearModel`, `glm_model`.
 """
-function glm_report(fitresult)
-    deviance = GLM.deviance(fitresult)
-    dof_residual = GLM.dof_residual(fitresult)
-    stderror = GLM.stderror(fitresult)
-    vcov = GLM.vcov(fitresult)
-    return (; deviance=deviance, dof_residual=dof_residual, stderror=stderror, vcov=vcov)
+function glm_report(glm_model, features)
+    deviance = GLM.deviance(glm_model)
+    dof_residual = GLM.dof_residual(glm_model)
+    stderror = GLM.stderror(glm_model)
+    vcov = GLM.vcov(glm_model)
+    coef_table = GLM.coeftable(glm_model)
+    # Update the variable names in the `coef_table` with the actual variable
+    # names seen during fit.
+    if length(coef_table.rownms) == length(features)
+        # This means `fit_intercept` is false
+        coef_table.rownms = string.(features)
+    else      
+        coef_table.rownms = [
+            (string(features[i]) for i in eachindex(features))...; "(Intercept)"
+        ]
+    end
+    return (; deviance, dof_residual, stderror, vcov, coef_table)
 end
 
```

Comment:
So for the moment, are we just throwing everything conceivable into the report, with a future PR to allow finer control (to suppress expensive computations)?

Reply:
Yeah, I still have a couple of PRs to push